web/models/user.ex

```diff
@@ -0,0 +1,29 @@
+defmodule MovieWagerApi.User do
+  use MovieWagerApi.Web, :model
+
+  schema "users" do
+    field :username, :string
+    field :profile_image_url, :string
+    timestamps()
+  end
+
+  @doc """
```

Comment:
Personal preference of mine but I vote in favor of removing boilerplate documentation as I don't think it adds value here.